module-apps/application-meditation/ApplicationMeditation.cpp

```diff
@@ -46,10 +69,29 @@ namespace app
 
     void ApplicationMeditation::createUserInterface()
     {
-        windowsFactory.attach(app::window::name::meditation_main_window,
-                              [](ApplicationCommon *app, const std::string &name) {
-                                  return std::make_unique<gui::MeditationWindow>(app);
-                              });
+        windowsFactory.attach(
+            app::window::name::meditation_main_window, [&](ApplicationCommon *app, const std::string &name) {
+                const auto durationInitValue = utils::getNumericValue<std::int32_t>(
+                    settings->getValue(settings::Meditation::duration, settings::SettingsScope::AppLocal));
+                const auto durationOnChangeCallback = [&](std::int32_t newValue) {
```

Comment:
```suggestion
                const auto durationOnChangeCallback = [this](std::int32_t newValue) {
```
Try to capture only what you use. This also applies to other places in PR :)

Reply:
done